source/agora/consensus/data/Block.d

```diff
@@ -459,13 +461,27 @@ public Block makeNewBlock (Transactions)(const ref Block prev_block,
     return block;
 }
 
-/// only used in unittests
+/// only used in unittests with defualt block signing with the genesis validators
 version (unittest)
 {
-    public Block makeNewBlock (Transactions)(const ref Block prev_block,
-        Transactions txs) @safe nothrow
+    public Block makeNewTestBlock (Transactions)(const ref Block prev_block,
```

Comment:
Why do you need `makeNewTestBlock`? It's just forwarding to `makeNewSignedTestBlock`. There's no use-case for it.

Reply:
I will remove it.